app-crypt/mc-agent/mc-agent-0.0.1.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+EGO_PN=github.com/raoulh/${PN}
+
+DESCRIPTION="Simple ssh-agent that loads keys stored from Moolticute"
+HOMEPAGE="https://github.com/raoulh/mc-agent"
+EGIT_COMMIT="29cf340ec129f5b70bc5d6f8abb9a53406d07305"
+SRC_URI="https://github.com/raoulh/mc-agent/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz
+	https://github.com/jawher/mow.cli/archive/2f22195f169da29d54624afd9eb83ada5c9e4ee9.tar.gz -> mow.cli.tar.gz
```

Comment:
Ok, I'm pretty sure this is going to be trouble. Distfile names must be unique throughout Gentoo (i.e. two different files must not use the same name). By using a generic name for a single commit of the package, you're just asking for somebody else to use the same name for a different commit (or package even, given the generic names).

That said, I suppose the eclass handles that somehow.

Reply:
this can be handled with golang-vcs-snapshot
here is an example:
```
EGO_VENDOR=(
    "github.com/jawher/mow.cli 2f22195f169da29d54624afd9eb83ada5c9e4ee9"
    "github.com/gorilla/websocket 21ab95fa12b9bdd8fecf5fa3586aad941cc98785"
    "github.com/satori/go.uuid 36e9d2ebbde5e3f13ab2e25625fd453271d6522e"
    "golang.org/x/crypto 2d027ae1dddd4694d54f7a8b6cbe78dca8720226 github.com/golang/crypto"
    )
....
SRC_URI="https://github.com/raoulh/mc-agent/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz
    ${EGO_VENDOR_URI}"
```